src/rust/node-identifier/src/lib.rs

```diff
@@ -112,7 +112,10 @@ where
                     Some(unid) => unid,
                     None => bail!("Could not identify ProcessNode"),
                 };
-                let session_db = SessionDb::new(self.node_id_db.clone(), "process_history_table");
+                let session_db = SessionDb::new(
+                    self.node_id_db.clone(),
+                    std::env::var("PROCESS_HISTORY_TABLE").expect("PROCESS_HISTORY_TABLE"),
```

Comment:
All these should also go in grapl-config, same comment about `docker-compose.yml`

Reply:
Resolved in newer commits